Sources/Clappr/Classes/Enum/Event.swift

```diff
@@ -53,4 +53,5 @@ public enum Event: String, CaseIterable {
     case didHideModal = "Clappr:didHideModal"
     case willShowModal = "Clappr:willShowModal"
     case willHideModal = "Clappr:willHideModal"
+    case didResize = "Clappr:didResize"
```

Comment:
Do we want to make the player listen and trigger this event like it does with `requestFullscreen event`? it would make easier for apps to listen. `player.on(Event.didResize)`

Reply:
and just reminding to include this new event in our doc = )

Reply:
Included.

For now, we will keep the `didResize` event only in Container.